samples/ChangefeedMigrationSample/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Changefeedconsole")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("HP Inc.")]
+[assembly: AssemblyProduct("Changefeedconsole")]
+[assembly: AssemblyCopyright("Copyright © HP Inc. 2017")]
```

Comment:
Where did you pick up this code?  It looks like at least this file is copyright by HP.  If that really is the case you need to create your own AssemblyInfo.cs file so we respect the copyright.